glide.yaml

```diff
@@ -0,0 +1,4 @@
+package: github.com/rigelrozanski/breaking
```

Comment:
oh whoops okay yeah - haven't done many of these PRs, I'll work out of a local CoinCulture instance here. Thanks